R-package/R/lgb.Dataset.R

```diff
@@ -1082,19 +1051,19 @@ dimnames.lgb.Dataset <- function(x) {
 #' labels <- lightgbm::get_field(dsub, "label")
 #' }
 #' @export
-slice <- function(dataset, ...) {
+slice <- function(dataset, idxset) {
```

Comment:
for reviewers wondering how it is ok to remove this `...` when it wasn't for `predict.lgb.Booster()` (https://github.com/microsoft/LightGBM/pull/4857#discussion_r762518752)... this situation is different because `slice()` is not a generic exported from base R.